ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py

```diff
@@ -163,7 +163,7 @@ def patch_column_description(
             None,
         )
 
-        if not col_index:
```

Comment:
I think we should keep this as `if not col_index`.

Reply:
@TeddyCr this condition becomes true for `col_index=0`, which causes an issue while further processing. That's why it needs to be checked with only `None`